CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Renamed existing `ptrace` wrappers to encourage namespacing ([#692](https://github.com/nix-rust/nix/pull/692))
+- `AioCb::Drop` will now panic if the `AioCb` is still in-progress ([#715](https://github.com/nix-rust/nix/pull/715))
+
+### Fixed
+- Fixed error handling in `AioCb::fsync`, `AioCb::read`, and `AioCb::write` ([#715](https://github.com/nix-rust/nix/pull/715))
```

Comment:
I think adding the specific thing that was fixed here would be good rather than having people see this and then having to reference the PR.